src/js/components/SidebarFilter.js

```diff
@@ -210,7 +210,7 @@ class SidebarFilter extends mixin(QueryParamsMixin) {
     let {props} = this;
 
     return (
-      <div className="side-list sidebar-filters hidden-medium hidden-small pod flush-top flush-left">
+      <div className="side-list sidebar-filters pod flush-top flush-left">
```

Comment:
I was able to remove these entirely because their parent, `ServiceSidebarFilters`, uses the appropriate class.